airflow/cli/commands/standalone_command.py

```diff
@@ -30,8 +30,9 @@
 from airflow.configuration import AIRFLOW_HOME, conf, make_group_other_inaccessible
 from airflow.executors import executor_constants
 from airflow.executors.executor_loader import ExecutorLoader
-from airflow.jobs.scheduler_job import SchedulerJob
-from airflow.jobs.triggerer_job import TriggererJob
+from airflow.jobs.job_runner import BaseJobRunner
+from airflow.jobs.scheduler_job import SchedulerJobRunner
```

Comment:
Yes. I plan to rename them to `_runner` as a follow-up PR. Just wanted to have a change that is super-easy to review and track changes fo - it's more difficult to review code that does both - changes and renames even if git is able to get some of those under control and get both modified and renamed files in most cases (but it is based on a heuristics and a number of changes, so it might go wrong because a number of changes is not small.